src/app/utils/util.ts

```diff
@@ -58,3 +58,29 @@ export function dedupePromise<T extends any[], K>(
 export function delay(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+/** Copy a string to the clipboard */
+export default function copyString(str: string) {
```

Comment:
Hmm, I see we removed the copy-string.ts file, wasn't able to find a reference to it being used though.

Reply:
It wasn't used anymore. I could remove it but I suspect we'll want it again soon.